src/wrap-store/wrapStore.js

```diff
@@ -99,7 +99,9 @@ export default (store, {
   /**
    * Setup external action handler
    */
-  chrome.runtime.onMessageExternal.addListener(dispatchResponse);
+  if (chrome.runtime.onMessageExternal) {
```

Comment:
Actually, on thinking more about `react-chrome-redux` specifically, I feel that it would be quicker (and simpler) at this point for us to use browser.* api with the [Firefox supplied polyfill for Chrome](https://github.com/mozilla/webextension-polyfill). I've been using this polyfill in my [extension project](https://github.com/AgrimPrasad/mozilla-observatory-extensions), and so far it has worked flawlessly in Chrome for me.

This would mean:

1. `react-chrome-redux` works out of the box with Firefox and Edge.
2. `react-chrome-redux` continues to work without issues in Chrome.

A separate library which assists in the porting of Chrome extensions to WebExtensions would still be useful for other projects which are more invested in the chrome.* namespace, but it seems like an unnecessary overkill for `react-chrome-redux` 

What do you guys think?